spring-cloud-task-docs/src/main/asciidoc/appendix-cloud-foundry.adoc

```diff
@@ -0,0 +1,11 @@
+
+[[appendix-cloud-foundry]]
+== Running a Task App on Cloud Foundry
+
+The simplest way to launch a Spring Cloud Task application as a task on Cloud Foundry
+is to use Spring Cloud Data Flow.  Via Spring Cloud Data Flow you can register your task application,
+create a definition for it and then launch it.  You then can track the task execution(s)
+via a RESTful API, the Spring Cloud Data Flow Shell, or the UI.  To learn out to get started
+follow the instructions in the
+https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#getting-started[Getting Started]
+section of the reference documentation.
```

Comment:
also add a pointer to the general flow of tasks https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#_the_lifecycle_of_a_task  will do it on merge.